src/app/instance.cc

```diff
@@ -80,12 +78,11 @@ void Instance::Initialize() {
 
 void Instance::Reload() {
   LOG(info) << "Instance: reloading";
-  // TODO(unassigned): reload kovri.conf
   // TODO(unassigned): locking etc.
   // TODO(unassigned): client/router contexts
-  m_IsReloading = true;
-  SetupTunnels();
-  m_IsReloading = false;
+  GetConfig().ParseKovriConfig();
```

Comment:
This doesn't work (`kovri.conf` re-loading). It won't work without work elsewhere (see other TODO's). Can you please remove this line or implement?